content/javascript/concepts/setTimeout/setTimeout.md

```diff
@@ -0,0 +1,81 @@
+---
+Title: 'setTimeout()'
+Description: 'Allows users to execute callbacks after a period of time expressed in milliseconds'
+Subjects:
+  - 'Web Development'
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Parameters'
+  - 'Arguments'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**`setTimeout()`** is a function serviced globally by the window object provided by the user's browser.
+
+## Syntax
+
+**`setTimeout()`** is capable of receiving multiple parameters where the first is a callback function, the second parameter receives a number that represents the time in milliseconds **(1s = 1000ms)**, that defines the time needed for the callback to execute, the third parameter onwards will be the parameters that the callback function would take in case arguments are defined within the callback.
```

Comment:
```suggestion
`setTimeout()` is capable of receiving multiple parameters where the first is a callback function, the second parameter receives a number that represents the time in milliseconds (1s = 1000ms), that defines the time needed for the callback to execute, the third parameter onwards will be the parameters that the callback function would take in case arguments are defined within the callback.
```
We only bold the first occurrence of the term.